daml-assistant/daml-helper/src/DamlHelper.hs

```diff
@@ -99,6 +99,10 @@ runDamlStudio replaceExt remainingArguments = do
         path = fromMaybe "." projectPathM
         command = unwords $ codeCommand : path : remainingArguments
     exitCode <- withCreateProcess (shell command) $ \_ _ _ -> waitForProcess
+    when (exitCode /= ExitSuccess) $
+        hPutStrLn stderr $
+          "Failed to launch Visual Studio Code." <>
+          " Take a look at https://code.visualstudio.com/Download for installation instructions."
```

Comment:
Is one reason it didn't start because it's not on their path? does a default install put it on the path? Is there any way to debug them failing for other reasons?